laravel-app/app/Http/Requests/UserPostRequest.php

```diff
@@ -6,6 +6,11 @@
 
 class UserPostRequest extends FormRequest
 {
+    /**
+     *
+     *
+     * @return array<string, mixed>
+     */
     public function rules(): array
```

Comment:
Since they are using Laravel 10. Laravel already removed doc block and used return type declaration.

Docs:
https://laravel.com/docs/10.x/releases#types
https://www.php.net/manual/en/language.types.declarations.php

Demo:
https://www.reddit.com/r/laravel/comments/11328e0/quick_demo_of_the_laravel_10x_shift/